.github/workflows/ci-frontend-deploy.yaml

```diff
@@ -0,0 +1,106 @@
+name: CI Frontend Deploy
+
+on:
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
+  workflow_dispatch: # manual trigger
+
+jobs:
+  # runs if CI workflow was successful OR if this was manually triggered
+  on-success:
+    runs-on: ubuntu-22.04
+    if: >
+      github.event_name == 'workflow_dispatch' ||
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          ref: main
+
+      - name: Set VERSION
+        run: echo "VERSION=$(./scripts/get_version.sh)" >> $GITHUB_ENV
+
+      - name: Heroku login
+        env:
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+        run: heroku container:login
+
+      - name: Release Backend on Heroku
+        uses: akhileshns/heroku-deploy@e3eb99d45a8e2ec5dca08735e089607befa4bf28
+        with:
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+          heroku_app_name: mitopen-rc
+          heroku_email: ${{ secrets.HEROKU_EMAIL }}
+          branch: main
+
+      - name: Write commit SHA to file
+        run: echo $GITHUB_SHA > frontends/main/public/hash.txt
+
+      - name: Build and push the NextJS Docker image
+        env:
+          HEROKU_APP_NAME: mitopen-rc-nextjs
```

Comment:
This seems like the wrong app name. Is it worth setting up that app in Heroku? Or should we do the work to just run that in K8s as well?

Reply:
oofff, I did not think of this.

Reply:
I would rather not go through getting this to run in k8s at the moment so I will take care of heroku today.